docs/core/actions.rst

```diff
@@ -12,7 +12,7 @@ Actions
 Actions are the things your bot runs in response to user input.
 There are four kinds of actions in Rasa:
 
- 1. **Utterance actions**: start with ``utter_`` and send a specific message
+ 1. **Response actions**: start with ``utter_`` and send a specific message
```

Comment:
@amn41 Should utterance actions be renamed like this?

Reply:
hmm good question. Would be more consistent, but then we'd probably have to change the classname and the `utter_` prefix. I'd say we keep calling them utterance actions. wdyt @abhilasharoy @tmbo ?

Reply:
I agree that they should be consistent, but I also think that we need a bigger discussion to make sure any affected pieces are changed accordingly (like the `utter_` prefix that @amn41 mentioned). 

So for now, I'd also vote to keep it as "utterance actions".

Reply:
Alrighty, I have reverted it to utterance actions.